deploy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ev
+echo `zip -r extension.zip extension/`
+echo `grunt`
```

Comment:
I'd personally prefer to keep it simple and move this to `.travis.yml`:

```
deploy:
  provider: script
  script: 'zip -r extension.zip extension/ && grunt'
  on:
    branch: master
    tags: true
```

What do you think?